src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/PreReleaseUserViewModels.cs

```diff
@@ -4,29 +4,21 @@
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
 {
-    public class PreReleaseUserViewModel
-    {
-        public PreReleaseUserViewModel(string email)
-        {
-            Email = email;
-        }
-
-        [EmailAddress] public string Email { get; }
-    }
+    public record PreReleaseUserViewModel(string Email);
```

Comment:
Don't we need the `[EmailAddress]` attribute on the `Email` param?

Reply:
I don't think so as I can't see anything using `PreReleaseUserViewModel` except for where it's a return type. Different request types are used for inviting users by email, and removing users by email.

Reply:
Ah cool, thought it might be used for a request, never mind!